src/client/datascience/kernel-launcher/kernelDaemon.ts

```diff
@@ -125,7 +129,10 @@ export class PythonKernelDaemon extends BasePythonDaemon implements IPythonKerne
                     // Don't call this.subject.error, as that can only be called once (hence can only be handled once).
                     // Instead log this error & pass this only when the kernel dies.
                     stdErr += out.out;
-                    traceWarning(`Kernel ${this.proc.pid} as possibly died, StdErr from Kernel Process ${out.out}`);
+                    // If we have requested for kernel to be killed, don't raise kernel died error.
+                    if (!this.killed) {
+                        traceWarning(`Kernel ${this.proc.pid} as possibly died, StdErr from Kernel Process ${out.out}`);
```

Comment:
We were unnecessarily throwing & logging errors when we were manually killing he kernels.